java-checks-test-sources/src/main/files/non-compiling/checks/SimpleStringLiteralForSingleLineStringsCheck.java

```diff
@@ -2,7 +2,7 @@
 
 public class SimpleStringLiteralForSingleLineStringsCheck {
   
-  public void str() { // Noncompliant@+1 [sc=23;endColumn=42]]{{Use simple literal for a single-line string.}}
+  public void str() { // Noncompliant@+1 [[sc=23;endColumn=43]]{{Use simple literal for a single-line string.}}
```

Comment:
wow, nice finding and undetected issue in precise location testing... 👍🏻 
worth a ticket to improve the verifier, don't you think?

Reply:
Was surprised too. Yep, we should definitely do sth with verify to detect such typos)